src/index.scss

```diff
@@ -32,4 +32,6 @@ body {
   height: calc(100% - 1px);
   display: flex;
   flex-direction: column;
+  overflow-y: scroll;
+  -webkit-overflow-scrolling: touch;
```

Comment:
just want to make sure this is behaving as expected on mobile. I know I had some issues getting zoom/scroll stuff working correctly in mobile safari, definitely found places where little changes would blow everything up.

Reply:
Tested on the about page, report incident page, and modal